index.js

```diff
@@ -89,6 +89,17 @@ const schemas = {
       'account_sid',
       'application_sid'
     ]
+  },
+  system_alerts: {
+    measurement: 'system_alerts',
+    fields: {
+      status: Influx.FieldType.STRING,
+      detail: Influx.FieldType.STRING,
+      host: Influx.FieldType.STRING
+    },
+    tags: [
+      'system_category'
```

Comment:
I would rather we named this "system_component".
Also, I would like to add an additional tag named "state", which will have a value of either 'offline' or 'online'.  That way we can also detect changes when a component goes offline or comes online.